### PR TITLE
introduction to security considerations as requirements, close #322

### LINKED ISSUE #322: security considerations are requirements on protocols
Many of the things that we are writing in the security considerations are really security requirements on implementations, rather than concerns with the architecture.

### PR DESCRIPTION
**Pull request by mcr**

### CODE REVIEW

File: draft-ietf-rats-architecture.md [hunk 1]
```diff
@@ -1286,6 +1286,11 @@ This approach is often called "Direct Anonymous Attestation".  See
 
 # Security Considerations {#security-considerations}
 
+This document provides an architecture for doing remote attestation.
+No specific wire protocol is documented here.
+Without a specific proposal to compare against, it is impossible to know if the the security  considerations listed below have been mitigated well.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1291]:
```suggestion
Without a specific proposal to compare against, it is impossible to know if the security considerations listed below have been mitigated well.
```
1) Fix double "the". 
2) Re "considerations ... mitigated":  you mitigate threats... are all security considerations threats?
3) some considerations aren't part of the implementation per se, like the discussion of physical security at a factory.  Would you consider those part of "a specific proposal"?

File: draft-ietf-rats-architecture.md [hunk 2]
```diff
@@ -1286,6 +1286,11 @@ This approach is often called "Direct Anonymous Attestation".  See
...
+The security considerations below should be read as being essentially requirements against implementations of the RATS Architecture.
```
Comment (dthaler) [inline draft-ietf-rats-architecture.md:1292]: Is physical security part of an "implementation"?

File: draft-ietf-rats-architecture.md [hunk 3]
```diff
@@ -1286,6 +1286,11 @@ This approach is often called "Direct Anonymous Attestation".  See
 
 # Security Considerations {#security-considerations}
 
+This document provides an architecture for doing remote attestation.
+No specific wire protocol is documented here.
+Without a specific proposal to compare against, it is impossible to know if the security considerations listed below have been mitigated well.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1291]:
```suggestion
Without a specific proposal to compare against, it is impossible to know if the security threats listed below have been mitigated well.
```

File: draft-ietf-rats-architecture.md [hunk 4]
```diff
@@ -1286,6 +1286,11 @@ This approach is often called "Direct Anonymous Attestation".  See
 
 # Security Considerations {#security-considerations}
 
+This document provides an architecture for doing remote attestation.
+No specific wire protocol is documented here.
+Without a specific proposal to compare against, it is impossible to know if the security threats listed below have been mitigated well.
+The security considerations below should be read as being essentially requirements against implementations of the RATS Architecture.
```
Comment (mcr) [inline draft-ietf-rats-architecture.md:1292]:
```suggestion
The security considerations below should be read as being essentially requirements against realizations of the RATS Architecture.
Some threats apply to protocols, some are against implementations (code), and some threats are against physical infrastructure (such as factories). 
```

**Pull request merged**